Breeding-Insight/Mgdb2BrapiV2Impl
# BI-1650 - Store and fetch genotype data

# Description
**Story:** https://breedinginsight.atlassian.net/browse/BI-1650

- Updated the creation of Call objects to add `genotypeMetadata` to `additionalInfo` for BrAPI 2.0 clients


# Dependencies


# Testing
- Verify that Gigwa starts up
- Verify that requesting for Calls returns `genotypeMetadata` in the `additionalInfo` field


# Checklist:
